src/rundeck/templates/job-configmap.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "rundeck.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: "{{ template "rundeck.name" . }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "rundeck.fullname" . }}-job
+data:
+  project.properties: |-

```

Comment:
Same as above: break files like this out into values.yaml.

Reply:
This files has been removed as part of cleanup

Reply:
link to code please. @Deepak-Routray ... I'm thinking it's probably in upstream, but please keep this PR in sync with what you push to upstream PR.  see the agreement.md in platform deploy.

Reply:
Refer new PR